### PR TITLE
Add pre-defined variant examples in representable

### PR DESCRIPTION
**Pull request by deepakmahakale**
### Motivation / Background

Add more examples with pre-defined variants to the documentation. 

### Detail

This pull request adds documentation to demonstrate the use of pre-defined variants

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] There are no typos in commit messages and comments.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Feature branch is up-to-date with `main` (if not - rebase it).
* [x] Pull request only contains one commit for bug fixes and small features. If it's a larger feature, multiple commits are permitted but must be descriptive.
* [ ] Tests are added if you fix a bug or add a feature.
* [ ] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.
* [x] PR is not in a draft state.
* [x] CI is passing.

### CODE REVIEW

File: activestorage/app/models/active_storage/blob/representable.rb [hunk 1]
```diff
@@ -17,6 +17,10 @@ module ActiveStorage::Blob::Representable
   #
   #   avatar.variant(resize_to_limit: [100, 100]).processed.url
   #
+  #   or if you are using pre-defined variants
```
Comment (p8) [inline activestorage/app/models/active_storage/blob/representable.rb:20]: Code examples have 2 extra spaces of indentation. As this is regular documentation, these spaces extra spaces should be removed:
```suggestion
  # or if you are using pre-defined variants
```

File: activestorage/app/models/active_storage/blob/representable.rb [hunk 2]
```diff
@@ -25,6 +29,10 @@ module ActiveStorage::Blob::Representable
   #
   #   <%= image_tag Current.user.avatar.variant(resize_to_limit: [100, 100]) %>
   #
+  #   or if you are using pre-defined variants
```
Comment (p8) [inline activestorage/app/models/active_storage/blob/representable.rb:32]: Similarly:
```suggestion
  # or if you are using pre-defined variants
```

**Review by p8 — CHANGES_REQUESTED** [review]
Thanks @deepakmahakale !
Blobs don't know about named variants, only Attachments do.
Could you move the documentation to app/models/active_storage/attachment.rb instead?

File: activestorage/app/models/active_storage/attachment.rb [hunk 3]
```diff
@@ -71,6 +87,14 @@ def preview(transformations)
 
   # Returns an ActiveStorage::Preview or an ActiveStorage::Variant for the
   # attachment with set of +transformations+ provided.
+  # Example:
+  #
+  #   avatar.representation(resize_to_limit: [100, 100]).processed.url
+  #
+  # or if you are using pre-defined variants
```
Comment (p8) [inline activestorage/app/models/active_storage/attachment.rb:94]:
```suggestion
  # or if you are using pre-defined variants:
```

File: activestorage/app/models/active_storage/attachment.rb [hunk 4]
```diff
@@ -60,6 +68,14 @@ def variant(transformations)
 
   # Returns an ActiveStorage::Preview instance for the attachment with the set
   # of +transformations+ provided.
+  # Example:
+  #
+  #   video.preview(resize_to_limit: [100, 100]).processed.url
+  #
+  # or if you are using pre-defined variants
```
Comment (p8) [inline activestorage/app/models/active_storage/attachment.rb:75]:
```suggestion
  # or if you are using pre-defined variants:
```

File: activestorage/app/models/active_storage/attachment.rb [hunk 5]
```diff
@@ -49,6 +49,14 @@ def purge_later
 
   # Returns an ActiveStorage::Variant or ActiveStorage::VariantWithRecord
   # instance for the attachment with the set of +transformations+ provided.
+  # Example:
+  #
+  #   avatar.variant(resize_to_limit: [100, 100]).processed.url
+  #
+  # or if you are using pre-defined variants
```
Comment (p8) [inline activestorage/app/models/active_storage/attachment.rb:56]:
```suggestion
  # or if you are using pre-defined variants:
```

**Review by p8 — CHANGES_REQUESTED** [review]
@deepakmahakale just some tiny nitpicks. Then it's good to go 🙂

**Comment by deepakmahakale** [comment]
@p8 Done.

**Comment by p8** [comment]
Thanks @deepakmahakale !

**Pull request merged**